pymediawiki/helpers.py

```diff
@@ -0,0 +1,58 @@
+import re
+import json
+import requests
+
+
+with open('constants.json') as _constants:
+    constants = json.load(_constants)
+
+def _strip_prop(text, prop):
+    return re.sub(prop+':', "", text, count=1)
+
+def _strip_JSON(res, prop, strip_chars, entry_prop):
+    ret = {}
+    for page_id, page_content in res['query']['pages'].items():
+        if prop not in page_content:
+            continue
+        ret[page_id] = [_strip_prop(entry[entry_prop], strip_chars) for entry in page_content[prop]]
+    return ret
+
+def _append_results(currlist, newlist, prop, strip_chars, entry_prop):
+    ret = _strip_JSON(newlist, prop, strip_chars, entry_prop)
+    for key in ret:
+        if key not in currlist:
+            currlist[key] = []
+        currlist[key] += ret[key]
+
+def _fetch_results(url, payload, headers, prop, strip_chars, entry_prop):
```

Comment:
Yeah, no problem. Let us go with the generic approach (the one returning dictionary) for now. The interface for single and multiple pages will need a little more thought.